README.md

```diff
@@ -465,6 +467,37 @@ extensions such as `import com.klaviyo.pushFcm.KlaviyoRemoteMessage.body` to acc
 We also provide an `Intent.appendKlaviyoExtras(RemoteMessage)` extension method, which attaches the data to your
 notification intent that the Klaviyo SDK requires in order to track opens when you call `Klaviyo.handlePush(intent)`.
 
+## In App Forms
+
+### Prerequisites
+- A published "in-app" form in the Klaviyo forms portal
```

Comment:
@ajaysubra should we include something like this?

Reply:
yeah I don't think this is needed. for instance we don't say having a push campaign is a prereq for setting up push. The audience here are other mobile devs so this prereq could just be what are the things needed for setting up IAFs on an android app.